packages/repo-tools/src/commands/generate-catalog-info/generate-catalog-info.ts

```diff
@@ -94,6 +99,28 @@ export default async (opts: CreateFixPackageInfoYamlsOptions) => {
     console.error();
     process.exit(1);
   }
+
+  if (checkForChanges) {
+    const instructions = results
+      .filter(isFulfilled)
+      .map(r => r.value)
+      .filter(r => r !== '');
+
+    // Non-empty instructions indicate changes would have been made.
+    if (instructions.length > 0) {
+      console.error(
+        '\ncatalog-info.yaml file(s) out of sync with CODEOWNERS or package.json\n',
+      );
+      console.error('Did you forget to run and commit the output of...');
+      console.error('  backstage-repo-tools generate-catalog-info');
+      console.error();
+      process.exit(1);
+    } else {
+      console.error(
```

Comment:
Console.log maybe?

Reply:
Don't remember where I picked this up, but I've always defaulted human-readable/diagnostic CLI output to `console.error()` / `stderr`, so that the "semantic" output of the command was free and clear on `stdout`, and could be easily accessed via pipes.

```sh
node -e 'console.error("This is diagnostic");console.log(`{"this":"is output"}`);' | jq
```

Reply:
Awesome; thanks for the clear feedback.